tests/cli/test_configure.py

```diff
@@ -23,3 +25,13 @@ def test_generate_config_called_with_mbed_os_path(self, program, generate_config
         CliRunner().invoke(configure, ["-m", "k64f", "-t", "gcc_arm", "--mbed-os-path", "./extern/mbed-os"])
 
         generate_config.assert_called_once_with("K64F", "GCC_ARM", program.from_existing())
+
+    @mock.patch("mbed_tools.cli.configure.generate_config")
+    @mock.patch("mbed_tools.cli.configure.MbedProgram")
+    def test_custom_targets_location_used_when_passed(self, program, generate_config):
+        program = program.from_existing()
+        custom_targets_json_path = pathlib.Path("custom", "custom_targets.json")
+        CliRunner().invoke(configure, ["-t", "gcc_arm", "-m", "k64f", "--custom-targets-json", custom_targets_json_path])
+
+        generate_config.assert_called_once_with("K64F", "GCC_ARM", program)
+        self.assertEqual(program.files.custom_targets_json, custom_targets_json_path)
```

Comment:
GitHub is showing removal of newline at the end of the file. Could you please add it back in commit "config: Handle --custom-targets-json"?